packages/ui/src/composables/useDeprecatedProps.ts

```diff
@@ -0,0 +1,16 @@
+import { __DEV__ } from '../utils/global-utils'
```

Comment:
Can we add this strategy to notion? "Deprecation guide" of sorts would be nice to have.